File: loianeTraining/javaBasico/exercicios-aulas-18-19/Ex35.java
public class Ex35 {
    public static void main(String[] args) {
        int[] a = { 1, 2, 3, 4, 5, 6, 8, 9, 10 };

        for (int i = 1; i <= a.length; i++) {
            System.out.println("\nDivisores de " + i);

            for (int j = 1; j <= i; j++) {
                if (i % j == 0) {
                    System.out.print(j + "\t");
                }
            }
        }
    }
}

File: loiane-training-csharp/procedural-poo/e/Ex35.cs
using System;

class Ex35
{
    static void Main(string[] args)
    {
        int[] vetorA = new int[10] { 12, 23, 34, 45, 56, 67, 31, 22, 6, 38 };

        for (int i = 0; i < vetorA.Length; i++)
        {
            Console.Write("\n\n{0}: ", vetorA[i]);

            for (int j = 1; j <= vetorA[i]; j++)
            {
                if (vetorA[i] % j == 0)
                {
                    Console.Write(j + " ");
                }
            }
        }
    }
}